### DISCUSSION TITLE
Chromdriver being detected despite using UC mode on Ubuntu 20.04.5 LTS

### DISCUSSION THREAD

**Discussion by iamumairayub** [seleniumbase/SeleniumBase#2311] [Q&A] [answered]
Here is my simple code where I am using SeleniumBase UC mode with a proxy.

```
driver = Driver(
    uc=True,
    headless=False,
    proxy='http://my-proxy'

)
driver.uc_open_with_reconnect("https://pixelscan.net/", reconnect_time=2)
driver.save_screenshot('screenshot.png')
```

But somehow Chromedriver is being detected.

![screenshot](https://github.com/seleniumbase/SeleniumBase/assets/13570295/cb14c72e-7b17-4609-8b39-89db4e4c8c3f)

Distributor ID: Ubuntu
Description:    Ubuntu 20.04.5 LTS
Release:        20.04
Codename:       focal

Python 3.10.7

seleniumbase.__version__
'4.21.4'

**Comment by mdmintz** [comment, marked as answer]
Your reconnect_time is too low. It takes several seconds to perform that check for that website.

Also, this is similar to https://github.com/seleniumbase/SeleniumBase/discussions/2246#discussioncomment-7502288 where the automation framework wasn’t detected, but something else was. Maybe you’re using a bad proxy too. But for sure, you need a longer disconnect there.
